docs/source/mda_COG.ipynb

```diff
@@ -0,0 +1,313 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.hist(diff.D.samples, density=True)\n",
+    "plt.axvline(diff.D.n, c='k')\n",
+    "plt.xlabel('$D$/cm$^2$s$^{-1}$')\n",
+    "plt.ylabel('$p(D$/cm$^2$s$^{-1})$')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Can you delete the final empty cell please.

Reply:
Done.